parser.y

```diff
@@ -2836,6 +2836,15 @@ DropTableStmt:
 	{
 		$$ = &ast.DropTableStmt{IfExists: true, Tables: $5.([]*ast.TableName), IsView: false}
 	}
+|	"DROP" "TEMPORARY" TableOrTables TableNameList RestrictOrCascadeOpt
```

Comment:
Can we have an `OptTemporary` just like MySQL?

https://github.com/mysql/mysql-server/blob/124c7ab1d6f914637521fd4463a993aa73403513/sql/sql_yacc.yy#L11702

Reply:
done.
And also using `IfExists`